src/main/scala/com/socrata/regioncoder/RegionCoderServlet.scala

```diff
@@ -1,124 +1,169 @@
 package com.socrata.regioncoder
 
-import javax.servlet.http.{HttpServletResponse => HttpStatus}
 import java.util.concurrent.ForkJoinPool
+import java.io.InputStreamReader
+import java.nio.charset.StandardCharsets
 
-import com.rojoma.json.v3.ast.{JObject, JString}
+import com.rojoma.json.v3.ast.{JValue, JObject, JString, JNull}
+import com.rojoma.json.v3.codec.JsonEncode
 import com.rojoma.json.v3.util.JsonUtil
 import com.socrata.geospace.lib.Utils._
 import com.socrata.regioncoder.config.RegionCoderConfig
 import com.socrata.soda.external.SodaFountainClient
-import org.scalatra.metrics.MetricsSupport
-import org.scalatra.{AsyncResult, BadRequest, Ok}
-import scala.concurrent.ExecutionContext
+import com.socrata.http.server.{HttpRequest, HttpResponse}
+import com.socrata.http.server.routing._
+import com.socrata.http.server.responses._
+import com.socrata.http.server.implicits._
+import com.rojoma.json.v3.interpolation._
+import org.slf4j.LoggerFactory
 
 // scalastyle:off multiple.string.literals
 class RegionCoderServlet(rcConfig: RegionCoderConfig, val sodaFountain: SodaFountainClient)
-  extends RegionCoderStack with RegionCoder with MetricsSupport {
-
-  // For FutureSupport / async stuff
-  implicit val executor = MDCHttpExecutionContext.fromThread(
-    ExecutionContext.fromExecutor(new ForkJoinPool(rcConfig.threadPoolLimit),
-                                  log("Uncaught exception", _)))
+  extends RegionCoder {
+  val logger = LoggerFactory.getLogger(classOf[RegionCoderServlet])
 
+  val concurrencyPerJob = rcConfig.threadPoolLimit
   val cacheConfig = rcConfig.cache
   val partitionXsize = rcConfig.partitioning.sizeX
   val partitionYsize = rcConfig.partitioning.sizeY
 
-  def pointcodeTimer[A](f: => A): A = timer("pointcode") {f}.call()
-  def stringcodeTimer[A](f: => A): A = timer("stringcode") {f}.call()
-  def debugGetTimer[A](f: => A): A = timer("debug-get") {f}.call()
-  def debugClearTimer[A](f: => A): A = timer("debug-clear") {f}.call()
+  def pointcodeTimer[A](f: => A): A = f //timer("pointcode") {f}.call()
```

Comment:
is this meant to be commented out?

Reply:
Dunno what it did before; it was a Scalatra thing.  I left them in so there's a single place to bring them all back if it turns out they're actually important for something.

Reply:
It looks like the MetricsSupport trait combined with this: https://github.com/socrata-platform/region-coder/blob/ff7b2626eef984af9ddb11b24218e711dc866ed5/src/main/scala/ScalatraBootstrap.scala#L44
provide the endpoints with the timer metrics

do we make use of that in any of out monitoring/graphs?

Reply:
For monitoring, not as far as I can tell.  I'm not sure where graphs would be configured, but it doesn't look like there's anything interesting going on with collectd in the container at least, which I'd expect if it were important for that.